src/extension.ts

```diff
@@ -232,6 +238,52 @@ export async function activate(context: vscode.ExtensionContext) {
 		},
 	);
 
+	// Component deletion context menu.
+	vscode.commands.registerCommand(
+		'apps-sdk.local-dev.delete-local-component',
+		// Delete folder will trigger file watcher
+		catchError('Delete local component', removeRecursively),
+	);
+
+	function parseComponentPath(path: string) {
+		const regex = new RegExp(`.*?/(${AppComponentTypes.join('|')})s/([^/]+)$`);
+		const matchComponentPath = path.match(regex);
+
+		if (matchComponentPath) {
+			const [, componentType, componentName] = matchComponentPath;
+			return { componentType: componentType as AppComponentType, componentName };
+		}
+		return null;
+	}
+
+	async function onFileDeleted(uri: Uri) {
```

Comment:
Uses the already existing import:

```suggestion
	async function onFileDeleted(uri: vscode.Uri) {
```